pytket/qir/conversion/gatesets.py

```diff
@@ -177,3 +177,38 @@ class CustomGateSet(NamedTuple):
     gateset={},
     tk_to_gateset=lambda optype: _TK_TO_PYQIR[optype],
 )
+
+PYQIR_FULL_GATESET = CustomGateSet(
```

Comment:
Why do we need this definition here, as it's redefined below?

Reply:
Removed in [35922b6](https://github.com/CQCL/pytket-qir/pull/229/commits/35922b6879a183f9ed219f3dc8f8f476937d676e)